pages/_app.jsx

```diff
@@ -1,7 +1,50 @@
+import { useEffect, useState } from "react";
 import "@/styles/globals.scss";
 import Header from "@/components/Header";
 
 export default function App({ Component, pageProps }) {
+	const [theme, setTheme] = useState("bright");
+	const [fontSize, setFontSize] = useState(16);
+
+	const brightPrimary = "#EC7373";
+	const brightSecondary = "#FFFFFF";
+	const brightText = "#000000";
+	const brightInvert = "0%";
+	const darkPrimary = "#B95151";
+	const darkSecondary = "#301E1E";
+	const darkText = "#FFFFFF";
+	const darkInvert = "100%";
+
+	useEffect(() => {
+		if (localStorage.getItem("font-size")) {
+			setFontSize(localStorage.getItem("font-size"));
+		} else {
+			localStorage.setItem("font-size", 16);
+		}
+
+		if (localStorage.getItem("theme")) {
+			setTheme(localStorage.getItem("theme"));
```

Comment:
don't know if it will be relavent after refactoring but you can also do:
```js
setTheme(localStorage.getItem("theme") || "bright");
```
instead of the condional, the upside, besides looks, is that here you access the localStorage only once

Reply:
doesnt work, says not defined

Reply:
It does work, which make sense:

```js
if(value) {
     set(value);
} else {
     set("default");
}
```

is equivalent to this:
```js
set(value || "default");
``` 

the `||` returns the second value if the first one is falsy

Reply:
noted, different usage